### ISSUE TITLE
Should selenized color syntax elements exactly like solarized?

### ISSUE THREAD

**Issue by jan-warchol** [jan-warchol/selenized#74]
Selenized builds on top of solarized by improving contrast and hues. What about token coloring choices? Should selenized mimic solarized as closely as possible, or could solarized choices be improved?

My opinion is that some choices in solarized are bad - in particular I dislike green keywords; to me they look ugly and also I think green is not strong enough for keywords:

![image](https://user-images.githubusercontent.com/1368508/77261534-67740000-6c8f-11ea-9815-5f733c361501.png)

I think that it's better to swap keywords with types:

![image](https://user-images.githubusercontent.com/1368508/77261535-6d69e100-6c8f-11ea-84ad-e58fc34fcb8f.png)

However, some users (e.g. @fladd) prefer to keep original coloring. I can do that if the majority of users think the same. Please give :+1: if you want selenized to be like solarized, or :-1: if you prefer a modified version.

**Comment by dngray** [comment]
Well with the `g:selenized_green_keywords=1` keyword you've achieved making both sets of people happy.

**Comment by fladd** [comment]
Could you also update the bw version? This one seems to have entirely different colours at the moment.

**Comment by jan-warchol** [comment]
> Could you also update the bw version? This one seems to have entirely different colours at the moment.

@fladd Umm, what do you mean? Black/white version is supposed to look different, like this:
![image](https://user-images.githubusercontent.com/1368508/78333378-051dd800-758a-11ea-9b76-f145961dea53.png)

Or did you mean to add this comment to https://github.com/jan-warchol/selenized/pull/76?

**Comment by fladd** [comment]
The colours for me are not the same as in the non-bw version. For instance, Python imports are blue all of a sudden, strings are green, functions are pink...
They should have the same colours than in the non-bw version, just with a different background, right?

**Comment by jan-warchol** [comment]
> The colours for me are not the same as in the non-bw version. For instance, Python imports are blue all of a sudden, strings are green, functions are pink...

Looks like the previous version, before #70 was merged. Are you sure you're using the latest code? Also, note that the directory layout changed in #73 and vim files are now in `editors/vim/` rather than just `vim/`.

> They should have the same colours than in the non-bw version, just with a different background, right?

Yes, almost the same. I mean, green will be a very slightly different green, but it still should be green.

**Comment by fladd** [comment]
Sorry, something went wrong with updating. Now the colours seem correct.

**Comment by rudock1** [comment]
Hi Mr Warchoł
I agree that it's better to swap keywords with types as you have shown in your second set of screen shots.  In a related way, on this page (https://github.com/jan-warchol/selenized/blob/master/balancing-lightness-of-colors.md) you show a sequence of screen shots using an older version of the 8-color-preview.sh test.  My question, however, is about which EDITOR and shell script COLOR THEME that was used to prepare these screen shots.  Here, too, the keywords are yellow not green.

I think it is most pleasing and I would love to see my own shell script work expressed with this color theme.  Could you please identify these components?  
Thanks in advance!
Billy

**Comment by patkru** [comment]
@rudock1 I don't think that this issue is the right place for your question, but to me it seems that on the screenshot you linked Jan used `vi` or `vim` (hint: sequence of spaces is red) and Selenized Dark theme

**Comment by kunthar** [comment]
Have you seen this?  
https://meat.io/oksolar

**Comment by braver** [comment]
I maintain a version of [Solarized for Sublime](https://packagecontrol.io/packages/Solarized%20Color%20Scheme) and have long ago [abandoned](https://github.com/braver/Solarized/issues/40#issuecomment-73196046) perfect compatibility with Ethan Schoonover's original implementations. Modern editors have so many features and more fine grained ability to highlight code, and support so many languages, that trying to maintain compatibility with screenshots from ages is both very hard and unsatisfying.
So, I took the same approach to my implementation of [Selenized for Sublime](https://github.com/braver/Selenized).
